html-practice/facebook-mock-up.html

```diff
@@ -0,0 +1,121 @@
+<!DOCTYPE HTML>
+<html>
+	<head>
+		<style>
+		html {
+			background-color: #e9eaed;
+		}
+		#Facebook-menu-bar {
+			background-color: #3333ff;
+			color: white;
+			position: relative;
+			top: 0;
+			height: 45px;
+			width: 100%;
+			margin: 0;
+		}
+		#search-bar {
+			position: relative;
+			top: 7px;
+			left: 225px;
+			width: 450px;
+			height: 15px;
+			padding: 5px;
+			border: 1px solid black;
+			border-radius: 5px;
+		}
+		main {
+			width: 500px;
+			height: 100%;
+			margin: 0 auto;
+			position: fixed;
+			left: 400px;
+		}
+		.status-box {
+			background-color: white;
+			border: 1px solid #e9eaed;
+			margin: 10px 0;
+			height: 140px;
+		}
+		.status-box-options {
+			color: blue;
+			border-bottom: 1px solid #e9eaed;
+			list-style-type: none;
+			display: inline-block;
+			width: 100%;
+			padding: 0;
+			margin: 0;
+		}
+		.status-box-options span {
+			float: left;
+			padding: 5px;
+			border-right: 1px solid #e9eaed;
+		}
+		.status-update-input img {
+			height: 50px;
+			width: 50px;
+			float: left;
+			margin-left: 5px;
+		}
+		.status-update-input input{
+			margin-left: 5px;
+			width: 80%;
+			padding: 15px;
+			border: none;
+			height: 40px;
+			float: left;
+		}
+		.status-update-options {
+			display: inline-block;
+			list-style-type: none;
+			width: 100%;
+			margin: 0;
+			padding: 0;
+			border-top: 1px solid #e9eaed;
+		}
+		.status-update-options span {
+			float: left;
+			padding: 5px;
+		}
+		.status-update-options input {
+			float: right;
+			margin: 5px;
+		}
+		</style>
+	</head>
+	<body>
+		<header id="Facebook-menu-bar">
+			<input type="text" id="search-bar" placeholder="Search Facebook">
+		</header>
+		<main id="content">
+			<div class="status-box">
+				<ul class="status-box-options">
+					<li><span>Update Status</span></li>
+					<li><span>Add Photos/Video</span></li>
+					<li><span>Create Photo Album</span></li>
+				</ul>
+				<div class="status-update-input">
+				<img src="logo.png"><input type="text" placeholder="What's on your mind?">
```

Comment:
Looks like this line needs tabbed over one. Also, I would put the image and input tags on separate lines.